autoload/ale/lsp.vim

```diff
@@ -253,6 +259,15 @@ function! ale#lsp#RegisterProject(conn, project_root) abort
         \   'init_request_id': 0,
         \   'message_queue': [],
         \}
+    " FIXME: What do we pass to ale#Var if we only want a global var
+    elseif ale#Var(-1, 'lsp_load_settings')
+        let l:settings_path = findfile(ale#Var(-1, 'lsp_settings_path'), a:project_root)
+        if empty(l:settings_path) || !filereadable(l:settings_path)
+            return
+        endif
+        let l:config = ale#util#FuzzyJSONDecode(readfile(l:settings_path), {})
+        " Queue up a workspace/didConfigurationChange message
+        call ale#lsp#Send(a:conn.id, ale#lsp#message#DidChangeConfiguration(l:config), a:project_root)
```

Comment:
This isn't the right function for this. This is just for setting up the project in the Dictionary. You might want `ale#linter#StartLSP` instead.